bin/alpen-bridge/src/mode/operator.rs

```diff
@@ -135,9 +136,15 @@ pub(crate) async fn bootstrap(params: Params, config: Config) -> anyhow::Result<
     )
     .await;
 
+    let current = bitcoin_rpc_client.get_block_count().await?;
+    let bury_height = current - config.btc_zmq.bury_depth as u64;
```

Comment:
use `checked_sub` instead of `-`: https://docs.rs/num/latest/num/trait.CheckedSub.html#tymethod.checked_sub

Already burn my hands with underflow/overflows in Rust...

Reply:
Went with saturating_sub instead.